src/pages/clients-list/index.js

```diff
@@ -11,19 +11,17 @@ import { formatDateDefault } from 'src/lib/date-helper'
 // ** Resources
 import { ResourceIds } from 'src/resources/ResourceIds'
 import { CTCLRepository } from 'src/repositories/CTCLRepository'
-import ErrorWindow from 'src/components/Shared/ErrorWindow'
 import { useWindow } from 'src/windows'
 import ClientTemplateForm from './forms/ClientTemplateForm'
 import { useResourceQuery } from 'src/hooks/resource'
+import Error from 'src/components/Shared/Error'
+import { useError } from 'src/error'
 
 const ClientsList = () => {
   const { stack } = useWindow()
-
-  //control
   const { getRequest } = useContext(RequestsContext)
-
-  //error
   const [errorMessage, setErrorMessage] = useState(null)
+  const { stack: stackError } = useError()
```

Comment:
Why do we have useError and Error used in this one?

Reply:
We will addressed the issue during the refactoring phase, but now the error window includes screen page only not all window.